### ISSUE TITLE
Why is  client.Games.List ids required now?

### ISSUE THREAD

**Issue by kaidoj** [Henry-Sarabia/igdb#47]
Hello.

IDs where nil and my script worked fine. Now the ids are required. Why?
My script is getting game realeases between two dates where ids are not existing. Is there other way i could achieve that after update.

**Comment by Henry-Sarabia** [comment]
Hey there.

Really sorry about that! The IGDB got a version update and I've been slowly migrating the package over to support it. There have been breaking changes, but they needed to happen to support the IGDB API V3.

As for your particular issue, I've split the List function into two distinct functions: List and Index. Before the update, List was performing two different functions based on whether a list of ID's were provided or not provided. Because of this, I split the latter functionality into its own function that is now called Index.

In summary, the Index function now provides the functionality you're looking for. And again, I'm really sorry for the inconvenience.

**Comment by kaidoj** [comment]
Np. Thank you for the package. I can just modify my script. I'll let u know if that solves my issue.

**Comment by Henry-Sarabia** [comment]
I'm just glad someone is getting some use out of it. I hope your problem gets resolved; if not, please let me know and I'll help you out in any way I can.

**Comment by kaidoj** [comment]
Got it working and it works fine. You can close the issue. 👍

**Comment by Henry-Sarabia** [comment]
Nice work and thanks for letting me know!
